src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/applyforcosts/DecideCostsHomeOfficePersonalisation.java

```diff
@@ -4,55 +4,51 @@
 
 import com.google.common.collect.ImmutableMap;
-import java.util.Arrays;
-import java.util.HashSet;
+import java.util.Collections;
 import java.util.Map;
 import java.util.Set;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCase;
-import uk.gov.hmcts.reform.iacasenotificationsapi.domain.entities.AsylumCaseDefinition;
 import uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.EmailNotificationPersonalisation;
 import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.CustomerServicesProvider;
 import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.EmailAddressFinder;
 import uk.gov.hmcts.reform.iacasenotificationsapi.infrastructure.PersonalisationProvider;
 
 @Service
-public class DecideCostsRespondentAndApplicantPersonalisation implements EmailNotificationPersonalisation {
+public class DecideCostsHomeOfficePersonalisation implements EmailNotificationPersonalisation {
 
-    private final String decideCostsNotificationForRespondentTemplateId;
+    private final String decideCostsNotificationTemplateId;
     private final String homeOfficeEmailAddress;
-    private final EmailAddressFinder emailAddressFinder;
     private final PersonalisationProvider personalisationProvider;
     private final CustomerServicesProvider customerServicesProvider;
 
-    public DecideCostsRespondentAndApplicantPersonalisation(
-        @Value("${govnotify.template.decideCostsApplication.respondent.email}") String decideCostsNotificationForRespondentTemplateId,
+    public DecideCostsHomeOfficePersonalisation(
+        @Value("${govnotify.template.decideCostsApplication.homeOffice.email}") String decideCostsNotificationTemplateId,
         @Value("${applyForCostsHomeOfficeEmailAddress}") String homeOfficeEmailAddress,
         EmailAddressFinder emailAddressFinder,
         CustomerServicesProvider customerServicesProvider,
         PersonalisationProvider personalisationProvider
     ) {
-        this.decideCostsNotificationForRespondentTemplateId = decideCostsNotificationForRespondentTemplateId;
+        this.decideCostsNotificationTemplateId = decideCostsNotificationTemplateId;
         this.homeOfficeEmailAddress = homeOfficeEmailAddress;
-        this.emailAddressFinder = emailAddressFinder;
         this.customerServicesProvider = customerServicesProvider;
         this.personalisationProvider = personalisationProvider;
     }
 
     @Override
     public String getTemplateId(AsylumCase asylumCase) {
-        return decideCostsNotificationForRespondentTemplateId;
+        return decideCostsNotificationTemplateId;
     }
 
     @Override
     public Set<String> getRecipientsList(AsylumCase asylumCase) {
-        return new HashSet<>(Arrays.asList(homeOfficeEmailAddress, emailAddressFinder.getLegalRepEmailAddress(asylumCase)));
```

Comment:
Do we have a separate personalisation for the applicant notification? As the ticket says the same notification to go out for HO and Applicant.